MAINTAINERS.md

```diff
@@ -1,6 +1,9 @@
 - [OpenSearch Security Maintainers](#opensearch-security-maintainers)
   - [Maintainers](#maintainers)
     - [Updating Practices](#updating-practices)
+- [Practices](#practices)
```

Comment:
> But this is a good topic to discuss on organization level.

@cliu123 Different repositories inside OpenSearch-Project will have different requirements to avoid all of those situations, I am tailoring this document around our repo first.